src/core/infrastructure/shortcut.d.ts

```diff
@@ -0,0 +1,26 @@
+import {Command} from './command';
+
+export declare class Shortcut {
+
+	constructor(table: object, manager: object);
+
+	manager: object;
```

Comment:
manager should be CommandManager you can put decloration of CommandManager in infrastructure folder near shortcut